website/content/en/preview/getting-started/migrating-from-cas/_index.md

```diff
@@ -0,0 +1,326 @@
+---
+title: "Migrating from Cluster Autoscaler"
+linkTitle: "Migrating from Cluster Autoscaler"
+weight: 10
+---
+
+This guide will show you how to switch from the [Kubernetes Cluster Autoscaler](https://github.com/kubernetes/autoscaler) to Karpenter for automatic node provisioning.
+We will make the following assumptions in this guide
+
+* You will use an existing EKS cluster
+* You will use existing VPC and subnets
+* You will use existing security groups
+* Your nodes are part of one or more node groups
+* Your workloads have pod disruption budgets that adhere to [EKS best practices](https://aws.github.io/aws-eks-best-practices/karpenter/)
+* Your cluster has an [OIDC provider](https://docs.aws.amazon.com/eks/latest/userguide/enable-iam-roles-for-service-accounts.html) for service accounts
+
+This guide will also assume you have the `aws` CLI installed.
+You can also perform many of these steps in the console, but we will use the command line for simplicity.
+
+## Create IAM roles
+
+To get started with our migration we first need to create two new IAM roles for nodes provisioned with Karpenter and the Karpenter controller.
+
+To create the Karpenter node role we will use the following policy and commands.
+
+```bash
```

Comment:
Would it be possible to put the bash scripts in a file and use the script tag like: https://github.com/aws/karpenter/blame/e7f4183921fa172f2afb5c126eed164fca19ca08/website/content/en/preview/getting-started/getting-started-with-eksctl/_index.md#L52

It's gonna make it easier to ensure they work since we can execute them and also allows reuse.

Reply:
Just curious, how do you run them when you have to export env vars that are used in later scripts?

Reply:
You can just `source filename.sh` that way the env var remains there or you can call them like [this](https://github.com/aws/karpenter/blob/main/website/content/en/preview/getting-started/getting-started-with-eksctl/scripts/install.sh)